### ISSUE TITLE
TKLBAM inithook should preinitialize existing APIKey if available

### ISSUE THREAD

**Issue by lirazsiri** [turnkeylinux/tracker#232]
Sometimes turnkey-init is run manually to partially reconfigure an appliance. Where possible the values should be pre-filled in, especially for values that are hard to cut and paste such as the APIKey for TKLBAM.
##

**Comment by OnGle** [comment]
Had a look at this, couldn't come to any conclusions on how it should be done after looking at how the key was stored. Probably best left to alon.

**Comment by JedMeister** [comment]
Agreed. Thanks for having a quick look anyway... I'll reassign Alon and remove it from the v14.1 milestone
